src/app/app.component.ts

```diff
@@ -66,24 +75,32 @@ export class AppComponent implements OnInit, OnDestroy {
     this.unsubscribeAll();
   }
 
-  async fetchAndSetFrontendVersion() {
+  async fetchAndSetFrontendVersion(): Promise<void> {
     this.frontendVersion = await this.versionService.getFrontendVersion();
     this.titleService.setTitle(`Ladybug - v${this.frontendVersion}`);
   }
 
   subscribeToServices(): void {
-    this.newTabSubscription = this.tabService.openReportInTab$.subscribe((value: ReportData) => {
-      this.openReportInSeparateTab(value);
-    });
-    this.newCompareTabSubscription = this.tabService.openInCompare$.subscribe((value: CompareData) => {
-      this.openNewCompareTab(value);
-    });
-    this.closeTabSubscription = this.tabService.closeTab$.subscribe((value: CloseTab) => {
-      const tab: Tab | undefined = this.tabs.find((t: Tab) => t.id === value.id);
-      if (tab) {
-        this.closeTab(tab);
-      }
-    });
+    this.newTabSubscription = this.tabService.openReportInTab$.subscribe(
+      (value: ReportData) => {
+        this.openReportInSeparateTab(value);
+      },
+    );
+    this.newCompareTabSubscription = this.tabService.openInCompare$.subscribe(
+      (value: CompareData) => {
+        this.openNewCompareTab(value);
+      },
+    );
+    this.closeTabSubscription = this.tabService.closeTab$.subscribe(
+      (value: CloseTab) => {
+        const tab: Tab | undefined = this.tabs.find(
+          (t: Tab) => t.id === value.id,
+        );
+        if (tab) {
+          this.closeTab(tab);
+        }
+      },
+    );
```

Comment:
Veel stukken code zijn flink ingekort, ik zie dat er in de prittierrc niet is opgegeven wat de lengte van de regels zijn. De default is volgende mij 80, wat best wel smal is. Deze default stamt af van de terminals die vroeger een standaard breedte hadden. Bij het Frank!Framework gebruiken we 120 omdat de meeste schermen dit makkelijk kunnen laten zien.

Reply:
Ik vind het ook erg lastig reviewen dat nu alle bestanden veranderd zijn. Ik kan niet snel de zinvolle wijzigingen zien.

Reply:
Het is gelukt om de maximum regel lengte op 120 te zetten. Daarna heb ik ng lint --fix gedaan. Ik ga zo kijken hoe het er daarmee uitziet.

Reply:
Je ziet nog veel wijzigingen omdat ESLint eisen stelt aan de volgorde van methods. Eerste moeten public methods met een annotatie zoals @ViewChild. Daarna gewone public methods, dan protected methods en dan private methods.

Reply:
Ja, dat is fijn.